### ISSUE TITLE
alt="" is treated as valid alt text

### ISSUE THREAD

**Issue by adamraine** [dequelabs/axe-core#4711]
### Product

axe-core

### Product Version

4.10.2

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation
I expected axe to flag image elements which have `alt=""`

#### Actual
Axe does not flag image elements with `alt=""`

#### How to Reproduce
Generate an axe report of https://thelibre.news/would-you-pay-for-free-software/

Several images don't have any real alt text if you do the query `document.querySelectorAll('img[alt=""]')`, but axe does not flag them.

#### Additional context
https://github.com/GoogleChrome/lighthouse/issues/16346

**Comment by WilcoFiers** [comment]
Thank you for reporting this issue. I understand your expectation that axe should flag images with alt="", but I want to explain why this is working as intended. In HTML, alt="" explicitly indicates that the image is decorative and should be ignored by screen readers, which is the correct way to mark decorative images according to WCAG SC 1.1.1.

Axe-core cannot determine if an image is decorative or informative. There is no reliable way to detect this, therefor axe-core cannot fail informative images that are marked up as decorative or vice versa. This is an unavoidable limitation of the technology. Axe-core can automatically find 57% of WCAG 2.0 issues, meaning that 43% of such issues need to be tested for manually or though tools like axe extensions [Intelligent guided tests](https://www.deque.com/get-started-axe-devtools-browser-extension/).

**Comment by thibaultmol** [comment]
Fair enough, it's a shame but it's indeed hard (probably impossible) to create a good system to determine if an image is indeed informative or not.
Part of me was wondering if there was a way to determine it if it's inside an article body or something. But yeah
